lib/active_model_serializers/adapter/base.rb

```diff
@@ -26,9 +26,7 @@ def serializable_hash(_options = nil)
       end
 
       def as_json(options = nil)
-        hash = serializable_hash(options)
-        include_meta(hash)
-        hash
+        serializable_hash(options)
       end
```

Comment:
alias method should be fine here, unless we want to keep the `root` option here like in the am:s:j in rails